index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="top-nav">
+        <a href="#" class="logo-txt"> Makoji David </a>
+        <div class="menu">
+          <img src="./img/icons/hamburger.png" alt="logo-icon" class="logo-icon" />
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="jumbotron">
+        <h1 class="title">Onate Makoji David</h1>
+        <h3 class="subtitle">Software developer</h3>
+        <p class="description">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <hr />
+      </section>
+      <div class="links">
```

Comment:
You can use HTML semantic tag(section) here. Since this is a section of your page, you should always use semantic tags that are descriptive of the section itself.